FluidNC/src/Maslow/Maslow.cpp

```diff
@@ -131,7 +131,8 @@ void Maslow_::begin(void (*sys_rt)()) {
 
   currentThreshold = 1500;
   lastCallToUpdate = millis();
-  orientation = VERTICAL;
+  orientation = HORIZONTAL;
```

Comment:
What id the difference between orientation and Maslow_vertical?

Reply:
What

Reply:
There's also a variable called Maslow_vertical that I thought was a booliean to indicate if we're operating in horizontal or vertical mode

Reply:
I understand